packages/schema/src/model.ts

```diff
@@ -767,7 +759,19 @@ export class JupyterGISModel implements IJupyterGISModel {
   }
 
   toggleIdentify() {
-    this._isIdentifying = !this._isIdentifying;
+    if (this._currentMode === 'identifying') {
+      this._currentMode = 'panning';
+    } else {
+      this._currentMode = 'identifying';
+    }
+  }
```

Comment:
Do you think we can have something more extensible for future:

```javascript
setIdentifyMode(enabled: boolean): void {
  this.currentMode = enabled ? 'identifying' : 'panning';
}


toggleIdentify(): void {
  this.setIdentifyMode(this._currentMode !== 'identifying');
}
```

Reply:
I'm not sure, we can always work on it in the future :)